bitbots_head_mover/package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>bitbots_head_mover</name>
+  <version>1.0.0</version>
+  <description> The bitbots_head_mover handles the head movement.
+      The body behavior sends commands for different head movement
+      patterns. It can also use an action to look at a specific
+      point on the field, e.g. to track the ball.</description>
+  <maintainer email="valerie.bartel@uni-hamburg.de">Valerie Bartel</maintainer>
+  <maintainer email="info@bit-bots.de">Hamburg Bit-Bots</maintainer>
+  <author email="info@bit-bots.de">Hamburg Bit-Bots</author>
+
+  <license>MIT</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>backward_ros</depend>
```

Comment:
Should also depend on the parameter blackboard

Reply:
This is a part of bitbots_utils which is already included